README.md

```diff
@@ -209,7 +209,52 @@ anyio.run(main)
 > [!NOTE]
     Consider supporting the creator of [`asgi-lifespan`][asgi-lifespan] [Florimond Manca][florimondmanca] via GitHub Sponsors.
 
-## 6. Use Lifespan State instead of `app.state`
+## 6. Use `pytest-asyncio` fixtures to test your FastAPI and ASGI applications
+
+When writing fixtures for your FastAPI applications, you should use [pytest-asyncio](pytest-asyncio) to define fixtures. Besides, if your application has a lifespan, you should use [`asgi-lifespan`](asgi-lifespan) to run the lifespan events.
```

Comment:
That's not really my recommendation... 🤔

I recommend using `anyio` because you already have the dependency.

Reply:
And the `asgi-lifespan` is mentioned above, no need to mention twice.